__tests__/app.test.js

```diff
@@ -7,6 +7,21 @@ const app =require('../app');
 beforeEach(() => seed(testData));
 afterAll(() => db.end());
 
+describe('GET /api', () => {
+  test("Status 200, responds with endpoints data", () => {
+    return request(app)
+      .get("/api")
+      .set("Accept", "application/json")
+      .expect("Content-Type", /json/)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.endpoints).toBeInstanceOf(Object);
```

Comment:
Here you could just check the contents of the json file are received, these endpoint tests are mainly to test that the documentation is delivered on an endpoint